### DISCUSSION TITLE
[BUG] protection exception (16777924) using `--assert-call` when calling `msgpack.unpackb`

### DISCUSSION THREAD

**Discussion by VeNoMouS** [dashingsoft/pyarmor#1306] [Q&A]
```
INFO     Python 3.10.11
INFO     Pyarmor 8.2.4
INFO     Platform linux.x86_64
```

**Comment by jondy** [comment]
Check this to ignore problem modules or functions
https://pyarmor.readthedocs.io/en/stable/topic/bccmode.html#ignore-module-or-function

**Comment by VeNoMouS** [comment]
@jondy this is still an issue, even if disable bcc on that file , with assert-call enabled, it products the protection exception
